geode-tcp-server/src/distributedTest/java/org/apache/geode/distributed/internal/tcpserver/TcpServerProductVersionDUnitTest.java

```diff
@@ -251,17 +251,15 @@ private TcpClient getTcpClient() {
         TcpSocketFactory.DEFAULT);
   }
 
-  private int createLocator(VM memberVM, boolean usingOldVersion) {
+  private int createLocator(VM memberVM) {
+    final int port = AvailablePortHelper.getRandomAvailableTCPPort();
```

Comment:
One minor tweak might be to move this definition closer to the first point of use down on line 262.